src/caselawclient/models/documents/body.py

```diff
@@ -129,6 +131,23 @@ def enrichment_datetime(self) -> Optional[datetime.datetime]:
     def content_as_xml(self) -> str:
         return self._xml.xml_as_string
 
+    @cache
+    def content_as_html(self, image_base_url: Optional[str] = None) -> str:
+        """Convert the XML representation of the Document into HTML for rendering."""
+
+        html_xslt_location = os.path.join(os.path.dirname(os.path.realpath(__file__)), "transforms", "html.xsl")
```

Comment:
This can/will get out of sync with the MarkLogic XSLT which will be a cause of confusing issues.

Reply:
The plan is for MarkLogic to no longer handle XSLT once this work is completed, so hopefully the overlap period will be short!